src/main/scala/com/wavesplatform/Application.scala

```diff
@@ -179,7 +179,7 @@ class Application(val actorSystem: ActorSystem, val settings: WavesSettings, con
 
     rxExtensionLoaderShutdown = Some(sh)
 
-    UtxPoolSynchronizer.start(utxStorage, settings.synchronizationSettings.utxSynchronizerSettings, allChannels, transactions)
+    UtxPoolSynchronizer.start(utxStorage, settings.synchronizationSettings.utxSynchronizerSettings, allChannels, transactions, blocks)
```

Comment:
`blocks` are all blocks which come from peers, even (possibly) those which are invalid and hence will never be appended to the blockchain.

Reply:
Replaced with newBlocks